### DISCUSSION TITLE
Why people are talking about feed all of a sudden? Lmao

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#27781] [Other Feature Feedback, Questions, & Ideas]
That sounds strange, but i guess the feed is launched today to all the users haha

**Comment by MrSarno** [comment]
GitHub recently made the For You feed available to most users, in addition to the existing Following feed. :)
<img width="952" alt="Screenshot 2022-07-31 at 09 57 13" src="https://user-images.githubusercontent.com/32073555/182018748-6e3dd349-382d-4ad3-a70d-f58b30bf5570.png">

**Reply by NitinIsFit** [reply]
Maybe i was a beta tester, i have for you page available for myself a long time ago lol

**Comment by jfullstackdev** [comment]
nope, the filter of that `For You` which now you can filter what you want to see, is the latest update because of the magnitude of requests in the past

you can even disable it by unchecking all, or just see what you want to see 

![image](https://user-images.githubusercontent.com/47092464/182029787-c334e5ca-6e50-4e25-a288-1219dbdc0438.png)
